AUDIT.md

```
@@ -1,5 +1,8 @@
 # bun audit
 
+> [!NOTE]
+> This file is the canonical record of dependency-audit overrides/exceptions (`bun audit --audit-level high` runs in the pre-commit hook on canary). It intentionally exists even when no overrides are required, do not delete it.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)